Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,45 @@
+    "The value of the drag coefficient of the cylinder is defined as: \n",
+    " \n",
+    " $$ C_D~=~\\frac{F_D}{\\frac{1}{2} . \\rho_{ref} . V_{ref} ^2 . A_{proj}}. $$\n",
     "\n",
-    "$$\n",
-    "\\mathbf{R_{ext}} + \\mathbf{F_v} -\n",
-    "\\int \\mathbf{n}~p~dA +\n",
-    "\\int \\rho \\mathbf{g}~dV = \n",
-    "\\frac{\\partial}{\\partial t} \\int \\rho \\mathbf{V} dV +\n",
-    "\\int (\\mathbf{n}~.~\\rho \\mathbf{V}) \\mathbf{V} dA .\n",
-    "$$\n",
+    "In this equation $F_D$, the total drag force due to pressure and viscous forces within the flow acting on the cylinder's surface in the streamwise direction, is normalized with the available momentum within the undisturbed incoming flow that could be exerted on the cylinder. The value of this momentum is calculated using $\\\\rho_{ref}$ as the reference density of the flow, which is the density of the flow. $V_{ref}$ as the undisturbed streamwise flow velocity in the inlet and $A_{proj}$, which is the projected area of the cylinder surface. For a two-dimensional cylinder the projected area is equal to the diameter of the cylinder. This is the area where pressure and viscous forces are acting on the cylinder in the streamwise direction. It should be noted that the pressure and viscous forces in the perpendicular direction to streamwise direction cancel each other. Hence, the forces and areas that they act on does not have any contribution to the drag force and coefficient.\n",
     "\n",
+    "Users can calculate and report the value of the drag coefficient following the report/forces/wall-forces. Running this command in the console will ask for the direction of the force, which in this case is the x-direction. Users should note that this command uses default reference variables to calculate the force coefficients. Therefore, it should be confirmed that the reference values are set correctly. This setting can be done using report/reference-values path and set the corresponding reference values for each variable as mentioned above and discussed extensively in the Physics section. The following table provides a summary of comparison between the experimentally measured and numerically calculated drag coefficient values: \n",
     "\n",
-    "## Cite Source(s)\n",
-    "```\n",
-    "> For more details on the physics, theory and equation derivation please see chapter 9, section 9.2 of \"A Physical Introduction to Fluid Mechanics by Alexander J. Smits\" 2nd edition. [Download Book Here!](http://www.efluids.com/efluids/books/efluids_books.htm)\n",
-    "```\n",
+    "| Reynolds #     | $C_D$ Experiment| $C_D$ Numerical | Relative Error [%]|\n",
+    "    | :-------------:|:-------------:  | :-----:|:--------:|\n",
+    "    | 150             | 1.34            | 1.36   |1.35|\n",
+    "Table 1. – Comparison between the experimentally measured and numerically predicted cylinder drag coefficient.\n",
     "\n",
```

Comment:
Another approach for validating the unsteady simulations is to estimate the vortex frequency shedding based on the flow field conditions. Please take a look at that comparison and add that comparison to this sections as well.